doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -194,10 +199,23 @@ Modem libraries
 Event manager
 -------------
 
+* Removed:
+
+  * Event manager had been updated and does no longer use orphaned sections to keep information about event types, listeners and subscribers.
+    You can remove :kconfig:`CONFIG_LINKER_ORPHAN_SECTION_PLACE` from config file.
+
 * Updated:
 
   * Modified the sections used by the event manager. Stopped using orphaned sections. Removed forced alignment for x86. Reworked priorities.
 
+Event manager Profiler Tracer
+-----------------------------
+
+* Removed:
```

Comment:
```suggestion
* Updated:
```
Should move to updated section as we are not completely removing the kconfig option.